src/storage/src/hummock/validator.rs

```diff
@@ -52,10 +52,13 @@ pub async fn validate_ssts(task: ValidationTask, sstable_store: SstableStoreRef)
                 continue;
             }
         };
+
+        // TODO: to use `prefetch: false` after `prefetch` be supported by
+        // SstableIteratorReadOptions
         let mut iter = SstableIterator::new(
             holder,
             sstable_store.clone(),
-            Arc::new(SstableIteratorReadOptions { prefetch: false }),
+            Arc::new(SstableIteratorReadOptions::default()),
```

Comment:
Why not keep the original implementation if we anyway want `prefetch: false`

Reply:
> Why not keep the original implementation if we anyway want `prefetch: false`

Currently, this configuration does not take effect. I think it's easy to misunderstand the logic here when reading the code (for me), so I added comments and used default like any other file